tensorflow/python/layers/convolutional.py

```diff
@@ -810,6 +810,7 @@ def conv3d(inputs,
       bias_constraint=bias_constraint,
       trainable=trainable,
       name=name,
+      dtype=inputs.dtype.base_dtype,
```

Comment:
Check if this breaks backward compatibility please.

Reply:
I have added this string similarly to conv2d methods.
Tests are rewritten similarly to conv2d tests so compatibility is checked by existing tests. There are both fp16 and fp32 in https://github.com/opensourcemattress/tensorflow/blob/e5f6a30d7b9d95e52fad491b6f951eb71b54df9e/tensorflow/python/kernel_tests/conv_ops_3d_test.py#L56 
and they are used here https://github.com/opensourcemattress/tensorflow/blob/e5f6a30d7b9d95e52fad491b6f951eb71b54df9e/tensorflow/python/kernel_tests/conv_ops_3d_test.py#L94

Reply:
I haven't changed this main line https://github.com/opensourcemattress/tensorflow/blob/e5f6a30d7b9d95e52fad491b6f951eb71b54df9e/tensorflow/python/kernel_tests/conv_ops_3d_test.py#L83
It means that there is nothing to do.

Reply:
LGTM.